### DISCUSSION TITLE
项目延续接管

### DISCUSSION THREAD

**Discussion by langningchen** [langningchen/XMOJ-Script#590] [Announcements]
@boomzero @PythonSmall-Q 你们俩有谁想要接管这个项目的？等到 #582 结束或者到了 2023-10-04 ，这个项目就停止更新了，在其之前，我们会发布**第一个，也是最后一个正式版本**1.0。如果有人要接管，我会发布公告，并且替换所有 Git 仓库链接，同时我会 Acheive 这个仓库，接管者应当 fork 然后更改，我不会 Transfer Ownership 。如果没有人接管，那么这个项目就废止了，同时，我也会发布全站公告。同时， **自发布本公告后（发布时间2023-10-02 14:50），不再接受任何 Issues 或者与 #582 无关的 PR 。**

**Comment by PythonSmall-Q** [comment]
艹有人摆了

**Comment by PythonSmall-Q** [comment]
我接吧。我会把你们变成collab

**Comment by PythonSmall-Q** [comment]
但是actions我不会留

**Reply by langningchen** [reply]
什么意思？Actions全删掉？那自动化发布你也用不了了？

**Comment by langningchen** [comment]
我不用 callab

**Comment by PythonSmall-Q** [comment]
太浪费了

**Comment by PythonSmall-Q** [comment]
就是把你当吉祥物

**Reply by langningchen** [reply]
好吧

**Comment by PythonSmall-Q** [comment]
@langningchen 看issue

**Comment by PythonSmall-Q** [comment]
是老的问题

**Comment by PythonSmall-Q** [comment]
等一下，那个网站怎么办

**Comment by PythonSmall-Q** [comment]
莫非我还得换域名

**Comment by PythonSmall-Q** [comment]
那到时候接手的时候一起转一下哈

**Comment by langningchen** [comment]
网站估计不好转，特别是域名

**Comment by PythonSmall-Q** [comment]
看看吧

**Comment by PythonSmall-Q** [comment]
qwq这么好的项目不想废了

**Comment by langningchen** [comment]
所以让你接管啊

**Comment by PythonSmall-Q** [comment]
我接管了那env啥的得重来了

**Comment by PythonSmall-Q** [comment]
pictures还是用你的repo吧

**Comment by PythonSmall-Q** [comment]
~~学术模式不想搞了~~摆

**Comment by PythonSmall-Q** [comment]
整了半天发现样例复制不了了

**Comment by langningchen** [comment]
Pictures 也换成你的 Repo

**Comment by langningchen** [comment]
> 整了半天发现样例复制不了了

那你到时候改吧

**Comment by PythonSmall-Q** [comment]
那我开始换了。你到时候帮个忙搞一下cf的事情

**Comment by PythonSmall-Q** [comment]
球球了

**Comment by langningchen** [comment]
可以，我准备写公告

**Comment by langningchen** [comment]
@PythonSmall-Q 你微信能上线吗？有一些关于 Secret 的东西在 Public 这里说不行

**Comment by PythonSmall-Q** [comment]
邮件

**Comment by PythonSmall-Q** [comment]
微信在我木秦哪里

**Comment by PythonSmall-Q** [comment]
seanoj_noreply@yeah.net

**Comment by langningchen** [comment]
好的

**Comment by langningchen** [comment]
还是 hack chat 吧，右键有点复杂

**Comment by langningchen** [comment]
旧网址： https://hack.chat/?swxcln

**Comment by langningchen** [comment]
@PythonSmall-Q 你需要重新接管一下 https://web.xmoj-bbs.tech/ ，不然非开发者模式的人更新不了

**Comment by langningchen** [comment]
项目已被成功接管
